valohai_cli/packager.py

```diff
@@ -40,6 +41,12 @@
 PackageFileInfo = namedtuple('PackageFileInfo', ('source_path', 'stat'))
 
 
+class GitUsage(Enum):
+    NONE = 0,
+    GITIGNORE_WITHOUT_GIT = 1,
+    GIT_LS_FILES = 2,
```

Comment:
I doubt you wanted to make these 1-tuples? 😄 
```suggestion
    NONE = 0
    GITIGNORE_WITHOUT_GIT = 1
    GIT_LS_FILES = 2
```